app/uk/gov/hmrc/individualsdetailsapi/cache/ModifiedDetails.scala

```diff
@@ -16,12 +16,30 @@
 
 package uk.gov.hmrc.individualsdetailsapi.cache
 
-import play.api.libs.json.{Format, Json}
-
-import java.time.LocalDateTime
+import play.api.libs.functional.syntax.toFunctionalBuilderOps
+import play.api.libs.json.{Format, JsResult, Json, Reads, Writes, __}
+import play.api.libs.json._
+import java.time.{Instant, LocalDateTime, ZoneOffset}
 
 case class ModifiedDetails(createdAt: LocalDateTime, lastUpdated: LocalDateTime)
 
 object ModifiedDetails {
-  implicit val format: Format[ModifiedDetails] = Json.format
+
+  private val localDateTimeFormat: Format[LocalDateTime] = Format(
```

Comment:
After checking if there is a library version of this formats, noticed we have `uk.gov.hmrc.mongo.play.json.formats.MongoJavatimeFormats` that will give us the same.

However, that will require `Instant` type fields than `LocalDateTime`.
And, I think we should be fine to change it here. `ModifiedDetails` is a model purely based for TTL and not domain focussed. So, change is less intrusive. And, it gives us the added benefit of keeping things simple.